iron-list.html

```diff
@@ -1619,7 +1619,6 @@
       }
       var modelTabIndex, activeElTabIndex;
       var target = Polymer.dom(e).path[0];
-      var activeEl = Polymer.dom(this.domHost ? this.domHost.root : document).activeElement;
```

Comment:
this shouldn't be removed. It could be:  
```js
var itemsHost = this._userTemplate.parentNode.domHost;
var activeEl = Polymer.dom(itemsHost || document).activeElement;
```